File: Python/Medium/الأسهم المتكررة.py

# Creator: OmarAfet
# https://profile.satr.codes/OmarAfet/public/overview
# https://github.com/OmarAfet

def arrowDuplicates(word: str) -> str:
    table = [x.lower() for x in word]
    result = ''

    for i in table:
        if table.count(i) > 1:
            result += '<'
        else:
            result += '>'

    return result

File: challenges/JavaScript/2. Medium/الأسهم المتكررة.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function arrowDuplicates(word) {
	let table = word.toLowerCase().split("");
	let result = "";

	for (let i of table) {
		if (table.filter((x) => x === i).length > 1) {
			result += "<";
		} else {
			result += ">";
		}
	}

	return result;
}
